pdfbox/src/main/java/org/apache/pdfbox/pdmodel/font/TrueTypeEmbedder.java

```diff
@@ -136,13 +136,17 @@ public final void buildFontFile2(InputStream ttfStream) throws IOException
     /**
      * Returns true if the fsType in the OS/2 table permits embedding.
      */
-    private boolean isEmbeddingPermitted(TrueTypeFont ttf) throws IOException
+    protected boolean isEmbeddingPermitted(TrueTypeFont ttf) throws IOException
     {
         if (ttf.getOS2Windows() != null)
         {
             int fsType = ttf.getOS2Windows().getFsType();
-            if ((fsType & OS2WindowsMetricsTable.FSTYPE_RESTRICTED) ==
-                             OS2WindowsMetricsTable.FSTYPE_RESTRICTED)
+            // use two's complement arithmetic to check if fsType is a power of 2
+            boolean powerOfTwo = fsType != 0 && (fsType & (fsType - 1)) == 0;
```

Comment:
I think for this to work against these first four bits you first need to mask against `0xF` and then you can probably use the simpler modulus operator to determine whether or not it is "even" since the first bit is reserved.

Reply:
Nice catch.  I was just thinking about the 3 permission bits we care about, and not other potential higher bits like that bitmap one.  I will fix this.

Reply:
Fixed